src/rest.cpp

```diff
@@ -293,6 +295,58 @@ static bool rest_chaininfo(AcceptedConnection* conn,
     return true; // continue to process further HTTP reqs on this cxn
 }
 
+static bool rest_mempool_info(AcceptedConnection* conn,
+                              const std::string& strURIPart,
+                              const std::string& strRequest,
+                              const std::map<std::string, std::string>& mapHeaders,
+                              bool fRun)
+{
+    vector<string> params;
```

Comment:
You could (in the future) start adding std:: in front of such code, as that means we can more easily get rod of ``using namespace std;`` :).